hobbyloop-api/src/main/java/hobbyloop/backend/api/controller/util/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,23 @@
+package hobbyloop.backend.api.controller.util;
+
+import hobbyloop.backend.domain.EnumTypeBindingException;
+import hobbyloop.backend.domain.ExceptionStatus;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+@Slf4j
+@RestControllerAdvice
+public class GlobalExceptionHandler {
+
+    /**
+     * enum Type 불일치 Exception
+     */
+    @ExceptionHandler(EnumTypeBindingException.class)
+    protected ResponseEntity<ExceptionResponse> handle(EnumTypeBindingException error) {
+        log.error("handle error", error);
+        final ExceptionResponse response = ExceptionResponse.from(ExceptionStatus.ENUM_TYPE_NOT_FOUND);
```

Comment:
계속 재사용되는 ExceptionResponse 면 static final 로 멤버 변수로 두어도 될 것 같습니다. 
```suggestion
public class GlobalExceptionHandler {
        private static final ExceptionResponse response = ExceptionResponse.from(ExceptionStatus.ENUM_TYPE_NOT_FOUND);
        
        @ExceptionHandler(EnumTypeBindingException.class)
        protected ResponseEntity<ExceptionResponse> handle(EnumTypeBindingException error) {
     log.error("handle error", error);
     ... 
     return new ResponseEntity<>(response, ExceptionStatus.ENUM_TYPE_NOT_FOUND.getHttpStatus());
     }
```

Reply:
static final 멤버 변수로 상수화 완료했습니다.
추후에 멤버 추가가 될 것으로 예상되어 이름을 명시하여 상수화 하였습니다